src/main/java/gradle_clojure/clojure/tasks/ClojureCompileOptions.java

```diff
@@ -48,6 +53,7 @@ public void setAotCompile(boolean aotCompile) {
     this.aotCompile = aotCompile;
   }
 
+  @Internal
```

Comment:
Why `@Interal` vs `@Input` for this one? Seems like it changes behavior of the task.

Reply:
Yes, it should be `@Input`.